### DISCUSSION TITLE
Limits on the number of variables and constraints?

### DISCUSSION THREAD

**Discussion by srimaljq1** [coin-or/pulp#371] [Q&A]
What is your question
-----------------------

Hi 

Is there a limit (upper bound) to the number of variables and constraints that can be handled by PulP? 
Are we looking at 100s or 1000s or more?

Thanks in advance

**Comment by nformbiz** [comment]
The number of variables and constraints is more of a restriction of the solver than it is Pulp.  That said, for reference, I have a problem with 13000 variables and ~3000 constraints which the included CMD solver produces a solution in ~1 minute.

If you go to commercial solvers like Gurobi, they can handle millions of variables and constraints.
